src/cse.cpp

```diff
@@ -722,7 +722,7 @@ LOCAL INT cse3( INT argc, const char* argv[])
 	brDiscardable = FALSE; 	// TRUE to return bin res in discardable memory blocks as well as files
 #endif
 	InputFileName = NULL;	// input file name: none yet. public variable.
-	char inputFileNameBuf[ _MAX_PATH];		// input file name put here
+	char inputFileNameBuf[CSE_MAX_FILENAME];		// input file name put here
```

Comment:
You are changing from "path length" to "filename length". I think this is appropriate here.